sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -90,6 +90,21 @@ private[sql] case class JDBCRelation(
 
   override val schema: StructType = JDBCRDD.resolveTable(url, table, properties)
 
+  /**
+   * Check if [[JDBCRDD.compileFilter]] can accept input filters.
```

Comment:
Hm.. for this one i'm not too sure if the function is also wrapped with [[ ]]. I will look through Scala guidelines.